pytorch/buildspec-1-10-1-neuron.yml

```diff
@@ -1,7 +1,7 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK pytorch
-version: &VERSION 1.10.1
+version: &VERSION 1.10.2
```

Comment:
Please also change the name of this file to `buildspec-1-10-2-neuron.yml.

Reply:
Done